data/common.yaml

```diff
@@ -1 +1,56 @@
---- {}
+---
+profile_backup::common::packages:
+  "gcc-c++":
+  "libacl":
+  "libacl-devel":
+  "libzstd-devel":
+  "lz4-devel":
+  "openssl-devel":
+  #"xxhash-devel":
+  "python3":
+  "python3-devel":
+  "python3-pip":
+  "pkgconfig":
+    provider: "pip3"
+    require:
+      - "Package[python3-pip]"
+  "setuptools-scm":
+    provider: "pip3"
+    require:
+      - "Package[pkgconfig]"
+  "borgbackup":
+    provider: "pip3"
+    require:
+      - "Package[gcc-c++]"
+      - "Package[libacl-devel]"
+      - "Package[libzstd-devel]"
+      - "Package[lz4-devel]"
+      - "Package[openssl-devel]"
+      #- "Package[xxhash-devel]"
+      - "Package[pkgconfig]"
+      - "Package[setuptools-scm]"
+profile_backup::common::pip_config: "/etc/pip.conf"
+profile_backup::common::pip_proxy: ""
+
+profile_backup::server::additional_sshd_match_params:
+  PubkeyAuthentication: "yes"
+  AuthenticationMethods: "publickey"
+  Banner: "none"
+  MaxAuthTries: "6"
+  MaxSessions: "10"
+  X11Forwarding: "no"
+profile_backup::server::backup_directory: null
+profile_backup::server::clients: {}
+#profile_backup::server::clients:
+#  "example client 1":
+#    hostname: "client1.local"
+#    ip: "127.0.0.1"
+#    key: "AAAAB..."
+#    key_type: "ssh-rsa"
+profile_backup::server::gid: null
+profile_backup::server::groupname: "nobody"
+profile_backup::server::require_storage:
+  - "File[%{::profile_backup::server::backup_directory}]"
```

Comment:
What happens if this resolves to "File[null]"?

Reply:
It breaks. 😄 
I'm removing this for now anyway.